### PR TITLE
9.0.1 preparation

### PR DESCRIPTION
**Pull request by jospoortvliet**
* [x] Changelog minus some potential last-minute stuff @cmonteroluque 
* [x] Added links to last releases @msrex 
* [ ] add VM links @jnweiger

### CODE REVIEW

**Comment by RealRancor** [comment]
@jospoortvliet All links to the github issues are missing "owncloud".

e.g.:

``https://github.com/core/issues/22993``

which should be:

``https://github.com/owncloud/core/issues/22993``

File: page-changelog.php [hunk 1]
```diff
@@ -2,6 +2,168 @@
...
+	<li>In files view, the user that made the shares is not shown <a href="https://github.com/core/issues/23645">#23645</a></li>
+	<li>files:transfer-ownership command does not work in some scenarios with shares <a href="https://github.com/core/issues/23686">#23686</a></li>
+	<li>Sharing to AD group fails when not all users have logged in <a href="https://github.com/core/issues/22907">#22907</a></li>
+	<li>Case insensitive group sharing <a href="https://github.com/core/issues/23223">#23223</a></li>
+	<li>Position of edit options in sharing menu confuses user <a href="https://github.com/core/issues/18163">#18163</a></li>
+	<li>Cannot download shared albums that were shared via link <a href="https://github.com/core/issues/gallery/554">gallery/#554</a></li>
+	</li></ul>
+<li>CalDAV/CardDAV, Calendar and Contacts
+	<ul>
+	<li> Allow to migrate calendars of all users <a href="https://github.com/core/issues/23113">#23113</a></li>
```
Comment (RealRancor) [inline page-changelog.php]: Additional space after the ``<li>``

File: page-changelog.php [hunk 2]
```diff
@@ -2,6 +2,168 @@
...
+	<li>Link to the folder/file is broken/wrong in the email sent via internal share. <a href="https://github.com/core/issues/23197">#23197</a></li>
+	<li>Dont break when there is an invalid share <a href="https://github.com/core/issues/23252">#23252</a></li>
+	<li>Add webdav share-types property to fix favorites/tags share status icon <a href="https://github.com/core/issues/23384">#23384</a></li>
+	<li>In files view, the user that made the shares is not shown <a href="https://github.com/core/issues/23645">#23645</a></li>
+	<li>files:transfer-ownership command does not work in some scenarios with shares <a href="https://github.com/core/issues/23686">#23686</a></li>
+	<li>Sharing to AD group fails when not all users have logged in <a href="https://github.com/core/issues/22907">#22907</a></li>
+	<li>Case insensitive group sharing <a href="https://github.com/core/issues/23223">#23223</a></li>
+	<li>Position of edit options in sharing menu confuses user <a href="https://github.com/core/issues/18163">#18163</a></li>
+	<li>Cannot download shared albums that were shared via link <a href="https://github.com/core/issues/gallery/554">gallery/#554</a></li>
```
Comment (RealRancor) [inline page-changelog.php]: ``https://github.com/core/issues/gallery/554`` -> ``https://github.com/owncloud/gallery/issues/554``

**Comment by jospoortvliet** [comment]
thanks @RealRancor fixing it all now.

File: page-changelog.php [hunk 3]
```diff
@@ -2,6 +2,167 @@
...
+	<li>Dont break when there is an invalid share <a href="https://github.com/owncloud/core/issues/23252">#23252</a></li>
+	<li>Add webdav share-types property to fix favorites/tags share status icon <a href="https://github.com/owncloud/core/issues/23384">#23384</a></li>
+	<li>In files view, the user that made the shares is not shown <a href="https://github.com/owncloud/core/issues/23645">#23645</a></li>
+	<li>files:transfer-ownership command does not work in some scenarios with shares <a href="https://github.com/owncloud/core/issues/23686">#23686</a></li>
+	<li>Sharing to AD group fails when not all users have logged in <a href="https://github.com/owncloud/core/issues/22907">#22907</a></li>
+	<li>Case insensitive group sharing <a href="https://github.com/owncloud/core/issues/23223">#23223</a></li>
+	<li>Position of edit options in sharing menu confuses user <a href="https://github.com/owncloud/core/issues/18163">#18163</a></li>
+	<li>Cannot download shared albums that were shared via link <a href="https://github.com/owncloud/core/issues/gallery/554">gallery/#554</a></li>
```
Comment (RealRancor) [inline page-changelog.php]: core/issues/gallery -> gallery/issues

File: page-changelog.php [hunk 4]
```diff
@@ -2,6 +2,167 @@
...
+<li>Only remove avatars from the folder we store them in <a href="https://github.com/owncloud/core/issues/23641">#23641</a></li>
+<li>XCache isAvailable check fails in OC9 <a href="https://github.com/owncloud/core/issues/23653">#23653</a></li>
+<li>Use the shipped cacerts.pem instead of the global one <a href="https://github.com/owncloud/core/issues/23662">#23662</a></li>
+<li>RemoveMount should lock <a href="https://github.com/owncloud/core/issues/17144">#17144</a></li>
+<li>Current-user-principal returns incorrect href. <a href="https://github.com/owncloud/core/issues/23306">#23306</a></li>
+<li>Trying to activate app on production stable9, doesn’t work (dependency analyzer) <a href="https://github.com/owncloud/core/issues/23668">#23668</a></li>
+<li>User trapped in app management screen <a href="https://github.com/owncloud/core/issues/23482">#23482</a></li>
+<li>Read available l10n files also from theme folder <a href="https://github.com/owncloud/core/issues/23567">#23567</a></li>
+<li>CSS isn't loaded correctly in ownCloud 9 version of the player <a href="https://github.com/owncloud/core/issues/files_videoplayer/36">files_videoplayer/#36</a></li>
```
Comment (RealRancor) [inline page-changelog.php]: core/issues/files_videoplayer -> files_videoplayer/issues

**Comment by RealRancor** [comment]
@jospoortvliet Two more left. Havn't checked for spelling (as the list seems to be generated from github which might have typos in general).

File: page-changelog.php [hunk 5]
```diff
@@ -2,6 +2,167 @@
...
+<li>Missing document icons in filter views <a href="https://github.com/owncloud/core/issues/23044">#23044</a></li>
+<li>Remove disabled autocorrect for new file names <a href="https://github.com/owncloud/core/issues/23024">#23024</a></li>
+<li>Getting a 404 when calling url/owncloud; fine when calling url/owncloud/index.php <a href="https://github.com/owncloud/core/issues/22970">#22970</a></li>
+<li>Call to a member function getRequest() on null <a href="https://github.com/owncloud/core/issues/23031">#23031</a></li>
+<li>Windows Office files via SMB mount through WebDAV is prompting for authentication <a href="https://github.com/owncloud/core/issues/22596">#22596</a></li>
+<li>php fatal error during upgrade from 8.2.2.2 to 9.0.0.1 <a href="https://github.com/owncloud/core/issues/23020">#23020</a></li>
+<li>500 on PROPFIND with "oc:owner-display-name" <a href="https://github.com/owncloud/core/issues/23116">#23116</a></li>
+<li>Explicitly check for port <a href="https://github.com/owncloud/core/issues/23156">#23156</a></li>
+<li>PHP 5.4 Warning Wording Needs Updating <a href="https://github.com/owncloud/core/issues/1170">#1170</a></li>
```
Comment (RealRancor) [inline page-changelog.php]: @jospoortvliet Do you know where this issue / link is coming from? I havn't found any references to that wording and https://github.com/owncloud/core/issues/1170 is a quite old and unrelated issue.

Comment (LukasReschke) [inline page-changelog.php]: This is an enterprise issue, ref https://github.com/owncloud/enterprise/issues/1170

Comment (RealRancor) [inline page-changelog.php]: Ahh, thanks. Have removed that in https://github.com/owncloud/owncloud.org/pull/925 as CE users don't have access to that.

**Pull request merged**